src/scanner/images/skopeo.ts

```diff
@@ -1,12 +1,33 @@
 import * as fs from 'fs';
 import sleep from 'sleep-promise';
+import crypto from 'crypto';
 
 import { logger } from '../../common/logger';
 import { config } from '../../common/config';
 import * as processWrapper from '../../common/process';
 import * as credentials from './credentials';
 import { SkopeoRepositoryType } from './types';
 
+type ImageDigests = {
+  manifestDigest?: string;
+  indexDigest?: string;
+};
+
+type ImageManifest = {
+  mediaType: string;
+  manifests: Array<{
+    digest: string;
+    platform: {
+      architecture: string;
+      os: string;
+      variant: string;
+    };
+  }>;
+};
```

Comment:
nit: these could go in the `types` file in this directory. They are 'private' though, in the sense that they're not needed elsewhere, so up to you.